base/abstractarray.jl

```diff
@@ -842,6 +842,12 @@ Represents the array `y` as an array having the same indices type as `x`.
 """
 of_indices(x, y) = similar(dims->y, oftype(indices(x), indices(y)))
 
+
+"""
+    full(F)
+
+Reconstruct the matrix `A` from the factorization `F=factorize(A)`.
+"""
```

Comment:
If it were me I'd just leave it as-is, since that's a larger task than simply relocating the docstring.